src/applications/static-pages/facilities/vet-center/components/VAFacilityInfoSection.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import VAFacilityAddress from './VAFacilityAddress';
+import VAFacilityTitle from './VAFacilityTitle';
+import VAFacilityPhone from './VAFacilityPhone';
+
+function VAFacilityInfoSection(props) {
```

Comment:
**Todo:** `mainPhone` and `vaFacility` should be destructured here to make the code a little cleaner.

Reply:
Sounds good